available_images.md

```diff
@@ -142,27 +142,32 @@ SageMaker Framework Graviton Containers (SM support only)
 
 NVIDIA Triton Inference Containers (SM support only)
 ============================
-**Note**:
-
-1. The following versions of the 23.`<XY>` container are supported: `23.01, 23.02, 23.03, 23.05, 23.06, 23.07, 23.08, 23.09, 23.10 and 23.12`.
-2. SageMaker Triton Inference Container does not support Tensorflow1 as of version 23.05 onwards, as upstream Triton container does not support
-Tensorflow(v1) native backend from version 23.04 onwards.
-3. SageMaker Triton Inference Container does not ship with the FasterTransformer(FT) backend from version 23.06 onwards since the upstream FT library is undergoing re-structuring. It was previously available from versions v22.12 - v23.05, experimentally. This note will be updated after the restructuring is complete.
+**Versions 23.12 and onwards**:
+1. Starting version 23.12, onwards, the sagemaker tritonserver is available in a different set of accounts than the previous ones. The new accounts are now the same as other DLCs listed on this page at the top, making it easier to switch containers going forward. These accounts are listed in section '# Available Deep Learning Containers Images' above.
+2. They can now be obtained programmatically from the sagemaker python sdk as:
+```
+from sagemaker import image_uris
+uri = image_uris.retrieve(framework=triton_framework, region=region, version=version, instance_type=instance_type)
+```
+where, for example `framework="sagemaker-tritonserver"`, and `region="us-west-2"`, `version="23.12"`, `instance_type="ml.g5.12xlarge"`.
+3. Unavailable versions: `24.02`.
+
+**Versions prior to 23.12**:
+
+1. For versions prior to 23.12, the following 23.`<XY>` versions are available: `23.01, 23.02, 23.03, 23.05, 23.06, 23.07, 23.08, 23.09, 23.10 and 23.12`.
```

Comment:
This section is for versions prior to 23.12, but includes 23.12. Is this intentional?

Reply:
That is right, the version is available in either route (Neo accounts and DLC accounts). There are some additional versions available in both route, but for avoiding further confusion, we have kept 23.12 as the overlapping version.